frontend/src/components/browse/NoItemsFound.js

```diff
@@ -10,11 +10,12 @@ const useStyles = makeStyles((theme) => {
   };
 });
 
-export default function NoItemsFound(/*{ type }*/) {
+export default function NoItemsFound({ type }) {
   const classes = useStyles();
   return (
     <Typography component="h4" variant="h5" className={classes.infoMessage}>
-      No results.
+      Error trying to retrieve <strong>{type}</strong> data from the server. <br />
```

Comment:
I changed this to "no results." because this message also appears if your filter simply returns no results. In that case it would be misleading to say that there has been an error retrieving data from the server.

Reply:
Makes sense 👍 will restore!